configuration/rules-dsl.md

```diff
@@ -206,6 +206,40 @@ For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
 > The parser will not recognize the whole text correctly without quotes.
 
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+You can listen for triggers from specific *trigger channels* that are provided by some addons.
+You can decide whether you want to catch only a specific trigger or any.
+Here is the syntax for all these cases (parts in square brackets are optional):
+
+```java
+Channel "<triggerChannel>" triggered [<triggerEvent>]
+```
+
+`triggerChannel` is the identifier for this specific channel.
+For example, the [Amazon Dash Button Binding]({{base}}/addons/bindings/amazondashbutton/readme.html#channels) provides the "press" trigger channel.
```

Comment:
You can probably make the case without addressing a specific binding. The documentation should be universally valid, a specific example feels like something one would find in a Tutorial. Don't you agree?

Reply:
Sounds valid.
I will remove this sentence and the astro binding hint too.